src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/config/ResourceServerConfiguration.kt

```diff
@@ -40,4 +42,11 @@ class ResourceServerConfiguration {
 
     return http.build()
   }
+
+  @Bean
```

Comment:
Just a thought; I wonder if this is the type of thing we should contribute back to the main template project? It would then save all future projects having this picked up by AppSec and having to make the same change as us? 🤔 
Just a thought;